app/Components/ArtisCardsItems/ArtistCardItems.ts

```diff
@@ -0,0 +1,29 @@
+import { ArtistCardItemsInterface } from '../ArtistCard/interfaces/artist-card-items.interfaces';
+
+const ArtistCardItems: ArtistCardItemsInterface[] = [
```

Comment:
I dont think that artistcardsitems is component oplace it in app/Dummy/artists.ts

Reply:
can you please more specific what you mean ?